src/App.css

```diff
@@ -1,39 +1,63 @@
-.App {
-  text-align: center;
-}
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+  font-weight: normal;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+body {
+  padding: 0;
+  margin: 0;
+  background: linear-gradient(to right, #0c0f13, #1e1f1f);
+  overflow: hidden;
 }
 
-.App-header {
-  background-color: #282c34;
+.calculator-grid {
+  display: grid;
+  justify-content: center;
+  align-content: center;
   min-height: 100vh;
+  grid-template-columns: repeat(4, 100px);
+  grid-template-rows: minmax(120px, auto) repeat(5, 100px);
+}
+
+.calculator-grid > button {
+  cursor: pointer;
+  font-size: 2rem;
+  border: 1px solid white;
+  outline: none;
+  background-color: rgba(255, 255, 255, 0.7);
+}
+
+.calculator-grid > button:hover {
+  background-color: rgba(255, 255, 255, 0.9);
+}
+
```

Comment:
- [ ] As mentioned in the project requirement, your design should match exactly the design provided for this project. take a look in the table below:

|Your Design|Original Design|
|-------------|-----------------|
|![image](https://user-images.githubusercontent.com/71874425/198335685-fdb1325b-1e75-4ab0-9cbd-fe422242bd94.png)|![image](https://user-images.githubusercontent.com/71874425/198335542-6b050201-10fc-4d60-82da-6a7973bee732.png)|

Kindly, make sure that you follow the original design colors, size, and tips.